core/utils/poly-look/src/react-components/chipGroup/chipGroup.css

```diff
@@ -2,14 +2,14 @@
   height: 32px;
   margin: 4px;
   padding: 0 12px;
-  border: solid 1px var(--theme-primary-color);
+  border: solid 1px var(--theme-secondary-color);
   border-radius: 16px;
-  background-color: var(--theme-secondary-color);
-  color: var(--theme-primary-color);
+  background-color: var(--theme-primary-color);
```

Comment:
In other contexts, you're using `color-*-dark`. Why not do it here and use `color-theme-dark`?

Reply:
Dark is only there since it is still used by polyExplorer at some places. Will be refactored out. When we only have one pool for colors (like I'm establishing in poly-look) dark is not descriptive enough. dark-blue? grey?

Reply:
Well, the choice of words is not so important here as the choice of identifier, with `color` going first in other cases, while it's going `last` here. Again, there are three main issues in computing: identifiers and off-by-one problems.